h/session.py

```diff
@@ -12,6 +12,10 @@ def model(request):
     session['userid'] = request.authenticated_userid
     session['groups'] = _current_groups(request)
     session['features'] = features.all(request)
+    user = request.authenticated_user
+    if features.flag_enabled(request, 'sidebar_tutorial'):
+        if user and not user.sidebar_tutorial_dismissed:
+            session['show_sidebar_tutorial'] = True
```

Comment:
I think we should probably put this in a suitable namespace under settings rather than at the root of the session object. Perhaps "prefs" or "settings"?